scripts/tracking/benchmark.py

```diff
@@ -0,0 +1,45 @@
+from gluoncv.utils.metrics.tracking import OPEBenchmark
+from glob import glob
+import argparse
+import os
+from gluoncv.data.otb.tracking import OTBTracking as OTBDataset
+from multiprocessing import Pool
+from tqdm import tqdm
+
+parser = argparse.ArgumentParser(description='tracking evaluation')
+parser.add_argument('--tracker_path', '-p',type=str,help='tracker result path')
+parser.add_argument('--dataset', '-d', default='OTB2015',type=str,help='dataset name')
+parser.add_argument('--num', '-n', default=1, type=int,help='number of thread to eval')
+parser.add_argument('--tracker_prefix', '-t',type=str, help='tracker name')
+parser.add_argument('--show_video_level', '-s', dest='show_video_level',action='store_true')
+parser.add_argument('--test_dataset',type=str,help='test_json dataset dir')
+parser.set_defaults(show_video_level=False)
+args = parser.parse_args()
+
+def benchmark():
```

Comment:
What is the difference between `test.py` and `benchmark.py`? Is there a good reason we separate them?

Reply:
test.py output is *.txt, where one video corresponds to one txt,which save in 'result'.
benchmark.py is recording evaluation such as success or Precision, which according to output and label.
I think it's convenient to add SiamMask and Pyost does this，and it's OK if merge them.

Reply:
**test.py** output is *.txt, where one video corresponds to one txt,which save in 'result'.
benchmark.py is recording evaluation such as success or Precision, which according to output and label.
I think it's convenient to add SiamMask and Pyost does this，and it's OK if merge them.

Reply:
### one

Reply:
_`one`_

Reply:
_test.py_ output is some *.txt, where one video corresponds to one txt, which records some output.
_benchmark.py_  records some Evaluation  such as Success or Precision according to output txt and label.
I think it's convenient to add SiamRPN++ or SiamMask and pysot does it，and it's OK if merge them

Reply:
**test.py** output is some *.txt, where one video corresponds to one txt, which records some output.
**benchmark.py** records some Evaluation such as Success or Precision according to output txt and label.
I think it's convenient to add SiamRPN++ or SiamMask and pysot does it，and it's OK if merge them

Reply:
### test.py output is some *.txt, where one video corresponds to one txt, which records some output.
### benchmark.py records some Evaluation such as Success or Precision according to output txt and label.
I think it's convenient to add SiamRPN++ or SiamMask and pysot does it，and it's OK if merge them

Reply:
`test.py` output is some *.txt, where one video corresponds to one txt, which records some output.
`benchmark.py` records some Evaluation such as Success or Precision according to output txt and label.
I think it's convenient to add SiamRPN++ or SiamMask and pysot does it，and it's OK if merge them

Reply:
Ok, I understand. I think it's good to have both files, let's keep it as it is.